.gitignore

```diff
@@ -39,6 +39,7 @@ dev_output.txt
 tmpcompiledjs*
 uploads/*
 env/
+/oppia-env/*
```

Comment:
Where is it comping from and any specific reason for starting with `/`?

Reply:
@DubeySandeep Sorry this was committed by mistake as it is a local environment setup!
 The requested changes are done!
 Thanks!!